gm4_dangerous_dungeons/data/gm4_dangerous_dungeons/worldgen/structure_set/dungeons_upper.json

```diff
@@ -0,0 +1,38 @@
+{
+  "structures": [
+    {
+      "structure": "gm4_dangerous_dungeons:badlands/lower",
+      "weight": 2
+    },
+    {
+      "structure": "gm4_dangerous_dungeons:badlands/upper",
+      "weight": 3
+    },
+    {
+      "structure": "gm4_dangerous_dungeons:stone/default",
+      "weight": 2
+    },
+    {
+      "structure": "gm4_dangerous_dungeons:stone/ocean",
+      "weight": 2
+    },
+    {
+      "structure": "gm4_dangerous_dungeons:desert",
+      "weight": 3
+    },
+    {
+      "structure": "gm4_dangerous_dungeons:ocean",
+      "weight": 3
+    },
```

Comment:
This file doesn't exist

Reply:
Good catch, i forgot to update it in there when i split them